src/azure-cli/azure/cli/command_modules/resource/custom.py

```diff
@@ -1025,11 +1025,14 @@ def update_resource_group(instance, tags=None):
 
 
 def export_group_as_template(
-        cmd, resource_group_name, include_comments=False, include_parameter_default_value=False):
+        cmd, resource_group_name, include_comments=False, include_parameter_default_value=False, resource_ids=None, skip_resource_name_params=False, skip_all_params=False):
     """Captures a resource group as a template.
-    :param str resource_group_name:the name of the resoruce group.
-    :param bool include_comments:export template with comments.
+    :param str resource_group_name: the name of the resource group.
+    :param str resource_ids: a string containing space-separated resource ids to filter the export by. To export all resources, do not specify this argument or supply "*".
+    :param bool include_comments: export template with comments.
     :param bool include_parameter_default_value: export template parameter with default value.
+    :param bool skip_resource_name_params: export template and skip resource name parameterization.
+    :param bool skip_all_params: export template parameter and skip all parameterization.
```

Comment:
For these parameters, I will recommend you to add them as `storage_true` and give short name for them.

Reply:
Storage_true is to set those values as true by default right? By our default is false, should it be storage_false instead? I may be off here.

Reply:
done.